### DISCUSSION TITLE
Material Icons not loading

### DISCUSSION THREAD

**Discussion by CodyBontecou** [cypress-io/cypress#16160] [Component Testing]
I'm working with the dependency `"material-design-icons": "^3.0.1",` which is used within my component like so: 
`<i class="pl-3 pr-5 material-icons text-xl text-gray-500">room</i>`

What am I doing wrong? It is showing the text `room` rather than the icon as expected.

**Comment by CodyBontecou** [comment]
Turns out my tailwindcss isn't loading either.

**Reply by lmiller1990** [reply]
Hello. How are you loading these into your app? Maybe in `main.ts` (or some other file)? If you have a repository reproducing the issue, I can take a look.

You may need to import them into each test, for example, to make sure they are globally available. In many of my tests I do `import 'tailwindcss/dist/tailwind.css`' (or something like that).

**Reply by CodyBontecou** [reply]
Hey, @lmiller1990 thanks for the reply. Regarding Tailwindcss, @elevatebart was able to help me out with this [PR](https://github.com/CodyBontecou/nuxt-tailwindcss-cypress-component-test/pull/1). It turns out it's a NuxtJS specific [issue](https://github.com/nuxt/nuxt.js/issues/9190).

Do you think this is the same issue causing my icons from not showing?

**Reply by lmiller1990** [reply]
Perhaps - very hard to say without a repo to reproduce. It sounds like the correct CSS/JS assets are not loaded.
